integration_tests/CMakeLists.txt

```diff
@@ -82,11 +83,20 @@ macro(RUN)
                 set_tests_properties(${name} PROPERTIES WILL_FAIL TRUE)
             endif()
         elseif(KIND STREQUAL "c")
-            add_custom_command(
-                OUTPUT ${name}.c
-                COMMAND lpython --show-c ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py > ${name}.c
-                DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
-                VERBATIM)
+            if (custom_c_postprocessor)
+                add_custom_command(
+                    OUTPUT ${name}.c
+                    COMMAND lpython --show-c ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
+                            --custom-c-postprocessor ${CMAKE_CURRENT_SOURCE_DIR}/${custom_c_postprocessor} > ${name}.c
```

Comment:
I think as part of this change, let's not do `--show-c`, but rather save the result to a file ourselves, so no need for ">" which is quite fragile in build systems.

Reply:
I agree. Will do this on Thursday.